src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsCaseMapping.java

```diff
@@ -0,0 +1,153 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
+
+import org.springframework.beans.factory.annotation.Value;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.model.HearingWrapper;
+import uk.gov.hmcts.reform.sscs.model.SessionCaseCodeMapping;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.*;
+import uk.gov.hmcts.reform.sscs.model.single.hearing.CaseDetails;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Objects.nonNull;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.isYes;
+import static uk.gov.hmcts.reform.sscs.helper.mapping.HearingsMapping.getSessionCaseCode;
+
+@SuppressWarnings({"PMD.UnnecessaryLocalBeforeReturn"})
+// TODO Unsuppress in future
+public final class HearingsCaseMapping {
+
+    public static final String CASE_TYPE = "caseType";
+    public static final String CASE_SUB_TYPE = "caseSubType";
+
+    @Value("${exui.url}")
```

Comment:
I don't think @Value works like this on static fields - you need to provide a setter